contracts/DPoS.sol

```diff
@@ -121,14 +128,45 @@ contract DPoS is IDPoS, Ownable {
         registeredSidechains[_addr] = true;
     }
 
-    function initializeCandidate(uint _minSelfStake) external {
+    function initializeCandidate(uint _minSelfStake, uint _commissionRate, uint _rateLockEndTime) external {
         ValidatorCandidate storage candidate = candidateProfiles[msg.sender];
         require(!candidate.initialized, "Candidate is initialized");
+        require(_commissionRate <= 10000);
```

Comment:
avoid hard coding

Reply:
As I just mentioned during the meeting, this parameter will be updated to a configurable param in governance PR. I think we can leave it as it is in this PR.

Reply:
Should not put the number directly here, but I think this value can be a constant, no need for governance.